### ISSUE TITLE
[Bug]: Luminosity contrast ratio of keyboard focus indicator on 'Left navigation' controls is 2.604:1 which is less than 3:1 

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#24162]
### Describe the bug

Luminosity contrast ratio of keyboard focus indicator on 'Left navigation' controls is 2.604:1 which is less than 3:1 

**Actual Result:**
The color contrast ratio of the keyboard focus indicator for the 'Left navigation' controls is 1.252:1 which is less than 3:1 

**Expected Result:**
The color contrast ratio of the keyboard focus indicator on the 'Left navigation controls' should be greater than or equal to 3:1. 

### To Reproduce

1. Open the URL- [Samples / General - Localization ⋅ Storybook (mgt.dev)](https://mgt.dev/next/v3/?path=/story/samples-general--localization)
2. 'Microsoft Graph Toolkit Playground' Overview page will get displayed. 
3. Navigate to the 'Left rail navigation' region using tab key. 
4. Navigate on the controls using tab key. 
5. Run AI tool and observe the issue. 

Attachments:
![ScreenShot](https://github.com/storybookjs/storybook/assets/93735775/383dc8bd-bf80-4b73-aa55-f5f53eb2bc4b)

### System

```bash
OS Build: Windows 11 
Version: 22H2 (OS Build 22621.2134) 
Browser: Edge dev 
Browser Version 116.0.1938.76 (Official build) (64-bit)
```

### Additional context

Same issue is repro on Components / mgt-agenda / Properties - Preferred Timezone ⋅ Storybook

**Comment by chocoscoding** [comment]
@shilman i would like fix this

**Comment by shilman** [comment]
@chocoscoding Awesome! Thank you for helping make Storybook better! 🙏  Please check out [how to contribute](https://storybook.js.org/docs/react/contribute/how-to-contribute) in our docs and feel free to ask questions in `#contributing` in [our Discord](https://discord.gg/storybook).
